recipe/meta.yaml

```diff
@@ -9,35 +8,44 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 0dcd8baa8d62b0c1d118b399b2ddba3c4aff271d0d7a9e0d4c1681c79035bbc7
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
+  skip: true  # [py<38]
 
 requirements:
   host:
-    - python >=3.5
+    - python
     - pip
     - setuptools >=44
     - setuptools_scm >=3.4.3
     - toml
+    - wheel
   run:
-    - python >=3.5
+    - python
     - six
```

Comment:
I don't see `six` listed anywhere. Let me know if I'm missing that somewhere.  

```suggestion
```

Reply:
I'm not seeing it either.